src/Http/Organizer/GetOrganizerRequestHandler.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Organizer;
+
+use CultuurNet\UDB3\EntityNotFoundException;
+use CultuurNet\UDB3\EntityServiceInterface;
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\Request\RouteParameters;
+use CultuurNet\UDB3\Http\Response\JsonLdResponse;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Slim\Psr7\Factory\StreamFactory;
+
+class GetOrganizerRequestHandler implements RequestHandlerInterface
+{
+    private EntityServiceInterface $organizerService;
+
+    public function __construct(EntityServiceInterface $organizerService)
+    {
+        $this->organizerService = $organizerService;
+    }
+
+    public function handle(ServerRequestInterface $request): ResponseInterface
+    {
+        $routeParameters = new RouteParameters($request);
+        $organizerId = $routeParameters->getOrganizerId();
+
+        try {
+            $organizer = $this->organizerService->getEntity($organizerId);
+
+            return (new JsonLdResponse())->withBody(
+                (new StreamFactory())->createStream($organizer)
+            );
```

Comment:
I wonder if there's a less convoluted way to construct this. Maybe we should build custom tooling for it?

Reply:
I changed the improved the `JsonResponse` constructor and it now only does a `json_encode` on types different from `string`.